lib/panel-league/garbage.js

```diff
@@ -143,9 +144,90 @@ const releaseGarbage = ((state) => {
   slabsToRemove.forEach((slab) => garbage.splice(garbage.indexOf(slab), 1));
 });
 
+
+const comboToSlabs = ((args) => {
+  const {minWidth, state} = args;
+  let slabs = [];
+  let size = args.size;
+  let x = 0;
+  if (size < minWidth) {
+    size = minWidth;
+  }
+  if (size > state.width) {
+    let overflow = (size % state.width);
+    if (overflow === 0) {
+      overflow = state.width;
+    }
+    const breakSize = Math.floor(overflow / 2) + minWidth;
+    const extraSlabs = comboToSlabs({
+      size: size - breakSize,
+      minWidth,
+      state
+    });
+    slabs = slabs.concat(extraSlabs);
+    size = breakSize;
+  }
+  if (size === minWidth) {
+    let right = 0;
+    let left = 0;
+    state.garbage.forEach((slab) => {
+      if (slab.width === minWidth) {
+        if (slab.x === 0) {
+          ++left;
+        } else {
+          ++right;
+        }
+      }
+    });
+    if (left > right) {
+      x = minWidth;
+    }
+  } else if (size < state.width) {
+    const RNG = JKISS31.unserialize(state.RNG);
```

Comment:
I don't like how acronyms become words when cased. `URL` instead of `Url` in my opinion.